src/native/corehost/bundle/extractor.cpp

```diff
@@ -7,6 +7,10 @@
 #include "pal.h"
 #include "utils.h"
 
+#ifdef __sun
```

Comment:
Why do we need `alloca.h` just for __sun?

Reply:
alloca() is defined in that header. we did the same thing elsewhere, e.g. https://github.com/dotnet/runtime/blob/d05950ba1ad440855f9851857bbad6f9d1a34321/src/coreclr/gc/env/gcenv.base.h#L14-L16